integration/simulation/simulation_tester.go

```diff
@@ -47,3 +53,19 @@ func testSimulation(t *testing.T, netw network.Network, params params.SimParams)
 	t.Logf("Simulation results:%+v", NewOutputStats(&simulation))
 	netw.TearDown()
 }
+
+func waitForNodesReady(obsNodes []*host.Node) {
+	now := time.Now()
+	//n := obsNodes[0]
+	for _, n := range obsNodes {
+		for {
+			if n.IsReady() {
+				fmt.Printf("Node %d is Ready after %s\n", obscurocommon.ShortAddress(n.ID), time.Since(now))
+				break
+			}
+			fmt.Printf("Waiting on Node %d after %s \n", obscurocommon.ShortAddress(n.ID), time.Since(now))
+			time.Sleep(time.Second)
```

Comment:
@otherview Playing around with this locally, I think we can really cut down this sleep, by at least 10x.

Reply:
Reduced 👍